test/basic/acm/main.tf

```diff
@@ -0,0 +1,76 @@
+terraform {
+  required_providers {
+    aptible = {
+      source = "aptible.com/aptible/aptible-iaas"
+    }
+    aws = {
+      source = "hashicorp/aws"
+    }
+  }
+}
+
+variable "organization_id" {
+  type = string
+}
+
+variable "environment_id" {
+  type = string
+}
+
+variable "aptible_host" {
+  type = string
+}
+
+variable "subdomain" {
+  type = string
+}
+
+variable "domain" {
+  type = string
+}
+
+provider "aws" {
```

Comment:
is this provider used?

Reply:
ah no - this will be used for the validation but you're right it's not necessary yet. I'll remove it so the history is clear